modal/image.py

```diff
@@ -236,6 +239,23 @@ def _get_image_builder_version(server_version: ImageBuilderVersion) -> ImageBuil
     return version
 
 
+def _create_context_mount(docker_commands: list[str], ignore: Callable[[Path], bool]) -> _Mount:
```

Comment:
Nit: I like to use `Sequence` instead of list since it will statically prevent accidental mutation of the argument